ui/components/app/transaction-detail-item/transaction-detail-item.component.js

```diff
@@ -15,26 +16,32 @@ export default function TransactionDetailItem({
   detailTotal = '',
   subTitle = '',
   subText = '',
+  noBold = false,
```

Comment:
Can we change this to something more positive and meaningful, like `boldHeadings = true`?

Reply:
done in 20cb59dc4